platform/platform-app/pom.xml

```diff
@@ -394,28 +395,28 @@
         <cxf.servlet-api.version>1.0</cxf.servlet-api.version>
         <cxf.servlet-api-2.5.artifact>geronimo-servlet_2.5_spec</cxf.servlet-api-2.5.artifact>
         <cxf.servlet-api-2.5.version>1.1.2</cxf.servlet-api-2.5.version>
-        <cxf.slf4j.version>1.7.21</cxf.slf4j.version>
+        <cxf.slf4j.version>1.7.22</cxf.slf4j.version>
         <cxf.specs.jaxws.api.version>1.2</cxf.specs.jaxws.api.version>
-        <cxf.spring.version>${spring.version}</cxf.spring.version>
-        <cxf.spring.security.version>3.2.9.RELEASE</cxf.spring.security.version>
+        <cxf.spring.version>3.2.9.RELEASE</cxf.spring.version>
```

Comment:
We are not actually using these and they will be removed with https://codice.atlassian.net/browse/DDF-2996.

In the meantime, I will remove this change from this PR.